### PR TITLE
Season 2 updates

### PR DESCRIPTION
**Pull request by ntowle**
- Added avoidable damage from Tormented mobs
- Added some missing avoidable damage in MotS
- Removed unavoidable Swift Slice in SoA
- Added a lot of avoidable damage in ToP
- Removed unavoidable Echoes of Carnage in ToP (probably meant Echo of Battle, which was added)
- Added avoidable damage in HoA
- Tweaked which spells are considered reasonably unavoidable for tanks

### CODE REVIEW

**Comment by ntowle** [comment]
Fixes #45

**Comment by amki** [comment]
Many thanks for your large contribution. 
Can you tell me when it's finished so I can merge it?

**Comment by ntowle** [comment]
@amki I think it's probably done for now. But if I find any other discrepancies while reviewing logs I can add to a S3 + Taza'vesh update or just open another PR. Thanks for reviewing!

File: ElitismHelper.lua [hunk 1]
```diff
@@ -47,14 +52,16 @@ local Spells = {
 	[326022] = 20,      -- Acid Globule (Spinemaw Gorger)
 	[340300] = 20,		-- Tongue Lashing (Mistveil Gorgegullet)
 	[340304] = 20,		-- Poisonous Secretions (Mistveil Gorgegullet)
+	[340311] = 20,      -- Crushing Leap (Mistveil Gorgegullet)
 	[331743] = 20,		-- Bucking Rampage (Mistveil Guardian)
+	[331748] = 20,      -- Back Kick (Mistveil Guardian)
 	--id ? [340160] = 20,		-- Radiant Breath (Mistveil Matriarch)
 	
 	--id ? [323177] = 20,		-- Tears of the Forest(Ingra Maloch)
+	[323250] = 20,      -- Anima Puddle (Droman Oulfarran)
```
Comment (redlab) [inline ElitismHelper.lua]: Already there on line 50

Comment (ntowle) [reply ElitismHelper.lua]: thanks, fixed

**Comment by ntowle** [comment]
Hi @amki, as 9.2.0 is out now I updated the toc. If you have a moment to merge this I'd appreciate it - thanks!

**Comment by amki** [comment]
Merged it, thank you for your contribution <3

**Pull request merged**